### DISCUSSION TITLE
Welcome to Baksteen.Blazor.CefSharp Discussions!

### DISCUSSION THREAD

**Discussion by jpmikkers** [jpmikkers/Baksteen.Blazor.CefSharp#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sakib1361** [comment]
Hi, nice library. 
I have come to a project to use blazor webview on avalonia project. I need windows and mac support primarily. But maui is a bit overkill and we do have some features and publishing methods which are outside of MAUI scope. 
I have seen that you have created wpf--avalonia blazor webview and a cefsharp--wpf webview. 
I was wondering if possible, to use cefglue instead to support mac. Or use wkWebkit somehow to support mac and keep webview2 for windows. 
Cefglue now support avalonia controls directly and can show rendered web page within avalonia. Url and navigation works. not sure how to use with blazor though.  
Not sure how to use wkwebkit other than targeting maui mac environment. 

I was using microsugar avalonia webview but that became obsolete and seems github repo is removed too.

Can you please provide some advice on how I can proceed correctly to support mac. thanks.

**Reply by jpmikkers** [reply]
There are other people maintaining forks of Microsugar webview, for example: https://github.com/acard0/Avalonia.WebView/ . If you go to insights / network you can see other forks of that too.
